arm-relay/2017-04-01/swagger/relay.json

```diff
@@ -361,11 +370,23 @@
             "schema": {
               "$ref": "#/definitions/RelayNamespace"
             }
+          },
```

Comment:
is PATCH a long running operation, if so, please use "x-ms-long-running-operation: true"